src/index.ts

```
@@ -10,4 +10,12 @@ const server = createServer({ config, dbClients});
 
 server.listen(config.port, () => {
   console.log(`listening on ${config.port}`);
+});
+
+process.on('uncaughtException', (err: Error) => {
```

eek ur swallowing exceptions lol, I have a module for doing this kinda stuff that I'll dig up

---

That's actually noise... I was trying to figure something else out when I added those